src/modules/transaction/components/TransactionDetailRow/TransactionDetailRow.css

```diff
@@ -0,0 +1,167 @@
+  width: 100%;
+  position: relative;
+
+  & > div:first-child {
+    padding: 20px 22px 20px 0px;
+    color: var(--color-content-light);
+    font-weight: var(--font-weight-normal);
+    font-size: 14px;
+  }
+
+  & > .eventAction {
+    padding: 0px;
+  }
+
+  &::before {
+    content: '';
+    min-width: 100%;
+    position: absolute;
+    height: 1px;
+    border-top: 1px var(--color-platinum) solid;
+  }
+
+  &:first-of-type::before {
+    display: none;
+  }
+
+  & > div {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: center;
+    color: var(--color-content-light);
+    font-size: var(--paragraph-font-size-s);
+    font-weight: var(--font-weight-normal);
+  }
+
+  &:hover .pinWrapper {
+    display: block;
+  }
+}
+
+.text {
+  padding: 0px 16px 0px 5px;
+
+  &.capitalized span {
+    text-transform: capitalize;
+  }
+
+  & span {
+    width: 100%;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    font-weight: var(--font-weight-semi-bold);
+    font-size: var(--font-size-h6);
+  }
+}
+
+.expandToggle {
+  --content-font: 'basier-circle';
+
+  padding: 0px 5px !important;
+  max-width: fit-content !important;
+  min-width: fit-content !important;
+  text-decoration: none !important;
+
+  & span {
+    font-weight: var(--font-weight-semi-bold) !important;
+    font-size: var(--font-size-h6);
+    color: var(--color-ultramarine-blue);
+    font-family: var(--content-font);
+  }
+
+  &.collapsed span {
+    color: var(--color-blue-gray) !important;
+  }
+}
+
+.addressWrapper {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 16px 0px;
+
+  & > div {
+    display: flex;
+    flex-direction: column;
+
+    &:last-child {
+      margin-left: 8px;
+    }
+
+    & > .name {
+      @mixin headingSmall;
+
+      margin-top: 0;
+      margin-bottom: 3px;
+      color: var(--color-maastricht-blue);
+      font-family: var(--content-font);
+
+      & > span {
+        @mixin contentNormal;
+
+        margin-left: 10px;
+        color: black;
+      }
+    }
+
+    & > .address {
+      @mixin contentNormal;
+
+      margin: 0;
+      color: var(--color-slate-gray);
+    }
+  }
+}
+
+.statusWrapper {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.transactionStatus {
+  padding: 8px 13px;
+  border-radius: 6px;
+  line-height: 12px;
+  letter-spacing: 0.2px;
+  white-space: pre-wrap;
+  display: inline-block;
+  font-weight: var(--font-weight-semi-bold);
+  font-size: var(--paragraph-font-size-s);
+  text-transform: capitalize;
+
+  &.success {
+    color: var(--color-jade-green);
+    background-color: var(--color-jade-green-transparent);
+  }
+
+  &.pending {
+    &.light {
+      color: var(--color-ink-blue);
+      background-color: var(--color-active-tab);
+    }
+
+    &.dark {
+      color: var(--color-ultramarine-blue);
+      background-color: var(--color-cloud-burst);
+    }
+  }
+
+  &.fail {
```

Comment:
You can use variables and re-define them using 
```
:root [data-theme="dark"] {}
```

So you don't have to add `.dark` and `.light` class names

Reply:
done